ENIGMAsystem/SHELL/Universal_System/Extensions/FileManip/include.h

```diff
@@ -0,0 +1,28 @@
+/*
+
+ MIT License
+ 
+ Copyright � 2020 Samuel Venable
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ 
+*/
+
+#define OVERRIDE_SANDBOX

```

Comment:
What the hell is sandbox?

Reply:
Are you mad at me or something? What has gotten with you today? The drama never ends. I get 3 reviews in two minutes all 3 of them seem to be intentionally trying to get on my nerves rather than do anything productive. 

That said, the current functions are going to be sandboxed, as if you didnt know that, we had a large discussion about sandboxing the file system and this overrides that behavior by turning on the extension checkbox. Sheesh.